R/modify.R

```diff
@@ -187,7 +187,7 @@ modify_where <- function(.x, .where, .f, ..., .error_call = caller_env()) {
     .x[.where] <- map_vec(.x[.where], .f, ..., .ptype = .x)
```

Comment:
Can't really wrap this one in `no_zap()`, but I think that is fine. I wouldn't want to slow down `map_vec()` by trying to push `no_zap()` into it. The snapshot error is also pretty good